lib/msal-core/src/utils/WindowUtils.ts

```diff
@@ -140,19 +139,31 @@ export class WindowUtils {
         return adalFrame;
     }
 
+    /**
+     * @hidden
+     * Removes a hidden iframe from the page.
+     * @ignore
+     */
+    static removeHiddenIframe(iframe: HTMLIFrameElement) {
+        document.body.removeChild(iframe);
+    }
+
     /**
      * @hidden
      * Find and return the iframe element with the given hash
      * @ignore
      */
-    static getIframeWithHash(hash: string) {
-        return Array.from(document.getElementsByTagName("iframe")).find(iframe => {
+    static getIframeWithHash(hash: string): HTMLIFrameElement {
+        const iframes = document.getElementsByTagName("iframe");
+        const iframeArray: Array<HTMLIFrameElement> = Array.apply(null, Array(iframes.length)).map((iframe: HTMLIFrameElement, index: number) => iframes.item(index)); // eslint-disable-line prefer-spread
+
+        return iframeArray.filter((iframe: HTMLIFrameElement) => {
             try {
                 return iframe.contentWindow.location.hash === hash;
             } catch (e) {
                 return false;
             }
-        });
+        })[0];
```

Comment:
why the first element?

Reply:
`.find` returns a single element, but `.filter` returns an array, so this just gets the first (and only) iframe with the hash we're looking for.

Reply:
i.e. `.find` is functionality equivalent (in happy-path) to `.filter()[0]`